Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/onboarding/OnboardingActivity.kt

```diff
@@ -71,9 +75,12 @@ class OnboardingActivity : AppCompatActivity(), LifecycleObserver, HasAndroidInj
     }
 
     fun completeOnboarding() {
-        LocalData.isOnboarded(true)
-        LocalData.onboardingCompletedTimestamp(System.currentTimeMillis())
-        settings.lastChangelogVersion.update { BuildConfigWrap.VERSION_CODE }
+        onboardingSettings.onboardingCompletedTimestamp = timeStamper.nowUTC
+
+        settings.lastChangelogVersion.update {
+            BuildConfigWrap.VERSION_CODE
+        }
```

Comment:
what about `isOnboarded `? it is removed ,but no saved using the new class

Reply:
`isOnboarded` has been replaced with a pure getter based on `onboardingCompletedTimestamp != null` to create a unified datasource and remove desync opportunities